crates/types/src/app.rs

```diff
@@ -106,7 +103,7 @@ pub enum Permission {
 )]
 #[serde(deny_unknown_fields)]
 pub struct BlockInfo {
-    pub height: Uint64,
+    pub height: u64,
```

Comment:
The reason to use `Uint64` over `u64` is that for numbers that are too big, some JS runtimes will serialize them to scientific notation, e.g. `1.234e+45`.

But it's unlikely our block height will ever reach that size, so using a `u64` is ok and simplier.